shared/src/fixtures/arrow.rs

```diff
@@ -122,15 +122,15 @@ pub fn primitive_record_batch() -> Result<RecordBatch> {
         // Field::new("float16_col", DataType::Float16, false),
         Field::new("float32_col", DataType::Float32, false),
         Field::new("float64_col", DataType::Float64, false),
-        Field::new(
-            "timestamp_col",
-            DataType::Timestamp(TimeUnit::Second, None),
-            true,
-        ),
+        // Field::new(
```

Comment:
Why are these commented out?

Reply:
The tests don't pass bc duckdb infers some types differently vs. datafusion

Reply:
I think you should leave comments on each one describing that, these are in our `shared` crate so I think they should be carefully documented.